Language/Functions/Math/map.adoc

```diff
@@ -51,7 +51,7 @@ The `map()` function uses integer math so will not generate fractions, when the
 
 [float]
 === Returns
-The mapped value.
+The mapped value. Data type: `int`.
```

Comment:
```suggestion
The mapped value. Data type: `long`.
```

Confirm accuracy of information via the function's source code:

https://github.com/arduino/ArduinoCore-API/blob/1.3.1/api/Common.h#L167

Reply:
As I understand from the linked line you shared @per1234 , the `map()` function uses integer math, with input as the `long` datatype (not `int`). Is this correct?

Reply:
>with input as the `long` datatype

It is true that the data type of the parameters (the function's input) is `long`.

However, your proposal here is documenting the return type (the function's output). So the parameter types are not relevant here. The relevant part of the function signature is the `long` that comes before the function name:

```cpp
long map(long, long, long, long, long);
```

That is the return type.

There is a nice tutorial about functions here:

https://cplusplus.com/doc/tutorial/functions/